oak_runtime/src/lib.rs

```diff
@@ -633,6 +639,15 @@ impl Runtime {
         node_info.privilege.clone()
     }
 
+    fn get_node_name(&self, node_id: NodeId) -> String {
+        let node_infos = self.node_infos.read().unwrap();
+        node_infos
+            .get(&node_id)
+            .expect("Invalid node_id")
+            .name
+            .clone()
+    }
```

Comment:
One thing to note is that unlike all other methods, this isn't labeled by a comment. 
I'm happy to add one, but the code feels fairly self explanatory?

Reply:
Might be nice to add a comment just for consistency (so there's not a blank space on the [doc page](https://project-oak.github.io/oak/oak_runtime/doc/oak_runtime/struct.Runtime.html))?  (It could also include the word "copy" as a hint/reminder that this does allocation on every invocation).

(But fine by me as-is too)

Reply:
Makes sense. :) Didn't realize these got pulled to the doc page, that's kind of nice.